### PR TITLE
Change ReadmeTemplates to builder pattern, fix missing ``` lines

### PR DESCRIPTION
**Pull request by daschult**

### CODE REVIEW

**Comment by fearthecowboy** [comment]
# 🤖 AutoRest automatic feature coverage report 🤖
*feature set version 2.6.0*

## ❌️ General: 97%

<details><summary>13 out of 540 features are not covered by tests</summary><p>

[`additionalPropertiesSubclass`](https://github.com/Azure/autorest.testserver/search?q=additionalPropertiesSubclass), [`animalNotFoundError`](https://github.com/Azure/autorest.testserver/search?q=animalNotFoundError), [`expectedNoErrors`](https://github.com/Azure/autorest.testserver/search?q=expectedNoErrors), [`expectedPetHungryError`](https://github.com/Azure/autorest.testserver/search?q=expectedPetHungryError), [`expectedPetSadError`](https://github.com/Azure/autorest.testserver/search?q=expectedPetSadError), [`getArrayEnumValid`](https://github.com/Azure/autorest.testserver/search?q=getArrayEnumValid), [`getArrayStringEnumValid`](https://github.com/Azure/autorest.testserver/search?q=getArrayStringEnumValid), [`getDateTimeRfc1123MinUtc`](https://github.com/Azure/autorest.testserver/search?q=getDateTimeRfc1123MinUtc), [`intError`](https://github.com/Azure/autorest.testserver/search?q=intError), [`linkNotFoundError`](https://github.com/Azure/autorest.testserver/search?q=linkNotFoundError), [`putArrayEnumValid`](https://github.com/Azure/autorest.testserver/search?q=putArrayEnumValid), [`putArrayStringEnumValid`](https://github.com/Azure/autorest.testserver/search?q=putArrayStringEnumValid), [`stringError`](https://github.com/Azure/autorest.testserver/search?q=stringError)</p></details>

## ❌️ Azure: 96%

<details><summary>4 out of 122 features are not covered by tests</summary><p>

[`LROPostDoubleHeadersFinalAzureHeaderGet`](https://github.com/Azure/autorest.testserver/search?q=LROPostDoubleHeadersFinalAzureHeaderGet), [`LROPostDoubleHeadersFinalAzureHeaderGetDefault`](https://github.com/Azure/autorest.testserver/search?q=LROPostDoubleHeadersFinalAzureHeaderGetDefault), [`LROPostDoubleHeadersFinalLocationGet`](https://github.com/Azure/autorest.testserver/search?q=LROPostDoubleHeadersFinalLocationGet), [`PagingMultipleLRO`](https://github.com/Azure/autorest.testserver/search?q=PagingMultipleLRO)</p></details>

File: src/DSL/MarkdownBuilder.cs [hunk 1]
```diff
@@ -234,7 +234,7 @@ public void Console(params string[] commands)
         /// <param name="commands">The commands that will be output as individual lines within the console section.</param>
         public void Console(IEnumerable<string> commands)
         {
-            Block(commands);
+            Block("bash", commands);
```
Comment (kpajdzik) [inline src/DSL/MarkdownBuilder.cs]: Why this change?

Comment (daschult) [reply src/DSL/MarkdownBuilder.cs]: Before this change our generated console blocks in markdown files didn't have a language specified. I don't know if this will actually make a change in our console blocks, but I figured its better to have it there just in case the text coloring/highlighting improves in the future.
